src/entries/popup/pages/home/TokenDetails/TokenDetails.tsx

```diff
@@ -287,34 +304,86 @@ function MoreOptions({ token }: { token: ParsedUserAsset }) {
         <AccentColorProvider
           color={token.colors?.primary || token.colors?.fallback}
         >
-          {!isNative && (
+          {!hidden && (
             <DropdownMenuItem
-              symbolLeft="doc.on.doc.fill"
-              onSelect={() => copyAddress(token.address)}
+              symbolLeft="pin.fill"
+              onSelect={() => {
+                if (pinned) {
+                  removedPinnedAsset({ uniqueId: token.uniqueId });
+                  return;
+                }
+
+                addPinnedAsset({ uniqueId: token.uniqueId });
+              }}
             >
```

Comment:
Looks like the shortcut hint UI is missing here 
<img width="266" alt="Screenshot 2024-03-23 at 3 29 49 AM" src="https://github.com/rainbow-me/browser-extension/assets/4412473/af0c878f-369e-4adc-b7ec-89b5dff040ce">

Reply:
We're going to fix this in another ticket. Seems like there is a bug with `<DropdownItem />`